diff --git a/lib/commands/parser.js b/lib/commands/parser.js
index 8bc19b1a67966ac359336311454e2669511a5476..a4443e815c5125d0d8285bafaf22cb9726854ab1 100644
--- a/lib/commands/parser.js
+++ b/lib/commands/parser.js
@@ -16,7 +16,7 @@ var fs = require("fs");
 
 var path = require("path");
 
-var gitDiff = require("git-diff");
+var gitDiff = function() { return "diff hidden"; }
 
 var _require = require("../lib.js"),
     getFiles = _require.getFiles,
diff --git a/src/commands/parser.js b/src/commands/parser.js
index 188aa1683b6e365fb002c6b34ce032390cb6cff3..c2ef7cb1abaca58455bb380a87b4583476aa1127 100644
--- a/src/commands/parser.js
+++ b/src/commands/parser.js
@@ -1,6 +1,6 @@
 const fs = require("fs");
 const path = require("path");
-const gitDiff = require("git-diff")
+const gitDiff = () => { return "diff hidden"; }
 const {
   getFiles,
   getCleanTokens,
