diff --git a/dist/utils/scope.js b/dist/utils/scope.js
index 412115cd02e2fd6f5177a1e7095fcbe7728b39b1..ff89a2ac967490acde296508ce1273a753347f16 100644
--- a/dist/utils/scope.js
+++ b/dist/utils/scope.js
@@ -25,7 +25,9 @@ function getContractsInScope(item) {
             else {
                 for (const a of i.symbolAliases) {
                     // Delayed function call supports circular dependencies
-                    scope[(_a = a.local) !== null && _a !== void 0 ? _a : a.foreign.name] = (_b = importedScope[a.foreign.name]) !== null && _b !== void 0 ? _b : (() => importedScope[a.foreign.name]());
+                    scope[(_a = a.local) !== null && _a !== void 0 ? _a : a.foreign.name] = (_b = importedScope[a.foreign.name]) !== null && _b !== void 0 ? _b : (() => { 
+                        return importedScope[a.foreign.name] ? importedScope[a.foreign.name]() : undefined;
+                     });
                 }
             }
         }
